spring-jdbc/src/main/java/io/crate/spring/jdbc/samples/BlogPostSerializer.java

```diff
@@ -0,0 +1,49 @@
+package io.crate.spring.jdbc.samples;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+
+import org.springframework.lang.NonNull;
+
+import io.crate.spring.jdbc.samples.domain.BlogPost;
+
+public class BlogPostSerializer {
+
+    public Map<String, Object> serialize(@NonNull final BlogPost post) {
```

Comment:
The whole serialization could be taken care of Jackson automatically, then this whole code would become unnecessary.

Reply:
The frontend is using a somehow weired data model. I wanted to used POJOs and simply convert on serializing/deserializing